include/nuttx/elf.h

```diff
@@ -142,9 +142,9 @@ bool up_checkarch(FAR const Elf_Ehdr *hdr);
 
 #ifdef CONFIG_LIBC_ARCH_ELF
 int up_relocate(FAR const Elf_Rel *rel, FAR const Elf_Sym *sym,
-                uintptr_t addr);
-int up_relocateadd(FAR const Elf_Rela *rel,
-                   FAR const Elf_Sym *sym, uintptr_t addr);
+                uintptr_t addr, FAR void *arch_data);
```

Comment:
```suggestion
                   uintptr_t addr, FAR void *arch_data);
```

Reply:
The indent in the file is correct, unified diff renders this change wrong.

Try using split diff